src/modules/stackoverflow/stackoverflow.js

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useRef } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { setFieldValue, getResponse, setResponse } from './stackoverflow.actions';
+
+const StackOverflowBrowserComponent = () => {
+	const state = useSelector((state) => state.stackoverflow);
+	const { values, response } = state;
+	const dispatch = useDispatch();
+	const isInitialMount = useRef(true);
+
+	useEffect(() => {
+		if (isInitialMount.current) {
+		   isInitialMount.current = false;
+		} else {
```

Comment:
Dlaczego jest to takie istotne?

Reply:
@devmentor-pl ponieważ nie chciałam odpytywać API przy pierwszym montowaniu komponentu (a tak się dzieje przy useEffect()). Teraz widzę, że sprawę załatwiłaby walidacja pola wyszukiwania - jeśli puste, nic nie rób, jeśli wypełnione - odpytuj API po zmianie dla values.sortByDate.